module/actor/mortal-actor-sheet.js

```diff
@@ -0,0 +1,375 @@
+/* global DEFAULT_TOKEN, Dialog, duplicate, game, mergeObject */
```

Comment:
I did inherit mortalActorSheet from the Coterie Sheet : features, notes and equipement are still there. We just add stats and a few more things.